.github/workflows/deploy.yaml

```
@@ -0,0 +1,96 @@
+permissions:
+  contents: write        # Docker 이미지 푸시 및 배포 시 사용
+
+jobs:
+  CI-CD:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./
+    steps:
+      # 1) 소스 체크아웃
+      - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)